FinancialModelingPrepApi/Model/StatementAnalysis/FinancialGrowthResponse.cs

```diff
@@ -0,0 +1,78 @@
+using System;
+using System.Text.Json.Serialization;
+
+namespace MatthiWare.FinancialModelingPrep.Model.StatementAnalysis;
+
+public class FinancialGrowthResponse
+{
+    [JsonPropertyName("symbol")]
+    public string Symbol { get; set; }
+    [JsonPropertyName("date")]
+    public DateOnly Date { get; set; }
+    [JsonPropertyName("calendarYear")]
+    public string CalendarYear { get; set; }
+    [JsonPropertyName("period")]
+    public string Period { get; set; }
+    [JsonPropertyName("revenueGrowth")]
+    public double? RevenueGrowth { get; set; }
+    [JsonPropertyName("grossProfitGrowth")]
+    public double? GrossProfitGrowth { get; set; }
+    [JsonPropertyName("ebitgrowth")]
+    public double? EarningsBeforeInterestAndTaxes { get; set; }
+    [JsonPropertyName("operatingIncomeGrowth")]
+    public double? OperatingIncomeGrowth { get; set; }
+    [JsonPropertyName("newIncomeGrowth")]
```

Comment:
Some spelling mistakes. Consider generating this using https://json2csharp.com/

```suggestion
    [JsonPropertyName("netIncomeGrowth")]
```

Reply:
Can you point out the spelling mistakes? Everything seemed fine when I ran the tests?

Reply:
Never mind, seen it 😊